include/xsimd/types/xsimd_sse_double.hpp

```diff
@@ -633,6 +633,7 @@ namespace xsimd
             static batch_type select(const batch_bool_constant<value_type, Values...>& cond, const batch_type& a, const batch_type& b)
             {
 #if XSIMD_X86_INSTR_SET >= XSIMD_X86_SSE4_1_VERSION
+                (void)cond;
```

Comment:
Is it possible to remove `cond` from the signature instead? (like you did in `xsimd_avx_float`)